src/probes/telem_record_gen.c

```diff
@@ -187,6 +193,17 @@ int validate_opts(void)
                 return ret;
         }
 
+        /* Event id  */
+        if(opt_event_id) {
+                if (strlen(opt_event_id) != 32) {
+                        fprintf(stderr, "Error: event_id should have 32 characters\n");
+                        return ret;
+                }
+                if(opt_event_id[strspn(opt_event_id, alphab)] != 0) {
+                        fprintf(stderr, "Error: event_id contains characters out of range, valid range %s\n", alphab);
```

Comment:
I think I would say

    Error: eventid contains invalid character '%c' at position %d in '%s', valid characters are %s

and fill in the 4 parameters. Makes the user do less work.